### PR TITLE
Integration test collaborators

### PR DESCRIPTION
**Pull request by nimat1**
Added one CMS file for Integration test collaborators
Added passed Screen short
image

### CODE REVIEW

File: integration-test/tests/cms/collaborators.test.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {expect, test} from '@playwright/test';
+test('GET /api/cms/v1/collaborators returns correct data', async ({request}) => {
```
Comment (dricazenck) [inline integration-test/tests/cms/collaborators.test.ts:2]:
```suggestion

test('GET /api/cms/v1/collaborators returns correct data', async ({request}) => {
```

File: integration-test/tests/cms/collaborators.test.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+const response = await request.get('/api/cms/v1/collaborators');
+expect(response.status()).toBe(200);
```
Comment (dricazenck) [inline integration-test/tests/cms/collaborators.test.ts:4]: Please could you also adjust the code format.

Comment (nimat1) [reply integration-test/tests/cms/collaborators.test.ts:5]: Did the changes,
image

Comment (dricazenck) [reply integration-test/tests/cms/collaborators.test.ts:5]: You need to adjust the format, please compare with other files, also you can use IntelliJ autoformat, please give a try.

**Review by dricazenck — APPROVED** [review]

**Pull request merged**